drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -31,6 +31,90 @@
 #define z_apb4_prescaler(v) LL_RCC_APB4_DIV_ ## v
 #define apb4_prescaler(v) z_apb4_prescaler(v)
 
+/* Macro to check for clock feasability */
+/* It is Cortex M7's responsibility to setup clock tree */
+/* This check should only be performed for the M7 core code */
+#ifdef CONFIG_CPU_CORTEX_M7
+/* Define primary oscillator frequencies */
+/* Suppress the cast to uint32_t which */
+/* prevents from compare with #if > operator*/
+/* original defines in stm32h7xx_hal_conf.h*/
+#define HSI_FREQ 64000000UL	/* HSI_VALUE ((uint32_t)64000000) */
+#define HSE_FREQ HSE_VALUE	/* HSE_VALUE overridden by the build system without cast to uint*/
```

Comment:
The line is over 80 characters. Please test like this (run the script from zephyr folder):
> ./scripts/checkpatch.pl -g HEAD-7

Reply:
Done, moved the comments to the previous line.